Automattic/delphin
# Checkout: Name actions file to be more explicit

This small pull request just renames the file where are located Redux action creators for the checkout. There is indeed no reason any more to have it defined as the default target of the `app/actions` folder.
#### Testing instructions
1. Run `git checkout update/rename-checkout-actions` and start your server, or open a [live branch](https://delphin.live/?branch=update/rename-checkout-actions)
2. Open the [`Home` page](http://delphin.localhost:1337/)
3. Check that everything works as before
#### Reviews

@Automattic/sdev-feed
